invisible_cities/cities/detsim_test.py

```diff
@@ -0,0 +1,103 @@
+import os
+import shutil
+import numpy  as np
+import pandas as pd
+import tables as tb
+
+from .. cities.detsim  import detsim
+from .. core.configure import configure
+
+from invisible_cities.core  import system_of_units as units
+
+
+def test_detsim_contains_all_tables(ICDATADIR, config_tmpdir):
```

Comment:
probably you want to use `output_tmpdir` (yes, i know in other city tests is usually all placed in config one); also is better to name the output files differently so you are not overwriting the same file in one pytest run